e2e/full-stack.e2e.ts

```diff
@@ -161,6 +162,7 @@ export class FullStackChart extends TalisChart {
       replicas: 2,
       tlsDomain: `*.${domain}`,
       externalHostname: externalHostname,
+      additionalExternalHostnames: [additionalHostname],
```

Comment:
Just to be clear, this is because we need to alias additional hostnames in TARL eg stubservice?

Reply:
This is to allow for auto registration of tenants hostnames